.github/workflows/cloud-installpack-bk-trigger.yml

```diff
@@ -0,0 +1,33 @@
+name: check formatting
```

Comment:
nit: is this a check formatting workflow?

Reply:
Nope, completely missed that. Would it be easier to merge this then do another backport of the fix from `dev`?